### DISCUSSION TITLE
Car logo for Polestar

### DISCUSSION THREAD

**Discussion by stephanlarsson** [elax46/custom-brand-icons#426] [General]
Hi @rchiileea, Love the Polestar logo as it is today. But would it be possible to just to the text logo and the graphical logo as separate icons to use? I think you call them "alts".

Many thanks,
Stephan

**Comment by stephanlarsson** [comment]
![polestar](https://github.com/elax46/custom-brand-icons/assets/120938592/77346259-f152-4cc5-ac4d-e8c7a9ab068a)
![2560px-Polestar_Logo svg](https://github.com/elax46/custom-brand-icons/assets/120938592/4bd8fc5c-82dd-470d-8142-6879fc49edfb)

**Comment by stephanlarsson** [comment]
Included the separate logos above in SVG

**Comment by rchiileea** [comment]
please only post these request under issues, I will add one alt just the logo itself

![image](https://github.com/elax46/custom-brand-icons/assets/87599807/98a5c3f4-6d9c-4252-b58d-a0de05c1e79c)

**Reply by stephanlarsson** [reply]
Sorry, was unaware. Will do next time. And thank you.
